src/index.ts

```diff
@@ -77,8 +80,40 @@ async function activate(
   } catch (error) {
     console.error(`Failed to load settings for the Git Extension.\n${error}`);
   }
+  try {
+    serverSettings = await getServerSettings();
+
+    // Version validation
+    if (!serverSettings.gitVersion) {
+      throw new Error('git command not found');
```

Comment:
```suggestion
      throw new Error('Git command not found - please ensure you have Git > 2 installed');
```

Reply:
I updated the suggestion to keep `git` (lowercase) to be consistent with the command name.